tools/tokenizer/src/Tokenizer.java

```diff
@@ -365,6 +365,32 @@ private static Analyzer getSuitableAnalyzer(String language, String mode) throws
         return arr;
     }
 
+    private static char PRETOK_DELIMITER = ' ';

```

Comment:
For better generality, we can make it as a FLAG string to allow any characters as delimiter. For example, --delimiter="\t, "

Reply:
Will do! Do we want this to imply `-pretok`? i.e. will both flags be needed?

Reply:
Using one flag also works if you prefer.